hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlFileSystem.java

```diff
@@ -68,12 +69,16 @@
 
 import static org.apache.hadoop.fs.adl.AdlConfKeys.*;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
statics at the bottom; these should go into the same block as com.*, ideally in order.